modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -0,0 +1,266 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="minimum-required-gcp-permissions-ipi_{context}"]
+= Required GCP permissions for installer-provisioned infrastructure
+
+When you attach the `Owner` role to the service account that you create, you grant that service account all permissions, including those that are required to install {product-title}.
+
+The following permissions are required for provisioning GCP infrastructure for creating and deleting the {product-title} cluster.
```

Comment:
Suggest to insert a sentence before "The following permissions are required...": 
If your organization’s security policies require a more restrictive set of permissions, you can create a [custom role](https://cloud.google.com/iam/docs/creating-custom-roles) with the necessary permissions. The following permissions are required...

The same suggestion for the UPI part, thanks!

Reply:
Another suggestion is to replace "provisioning GCP infrastructure" with "installer-provisioned infrastructure", i.e. the whole sentence would turn into: 

The following permissions are required for installer-provisioned infrastructure for creating and deleting the {product-title} cluster.

Reply:
Similarly, for the UPI part, replace "provisioning GCP infrastructure" with "user-provisioned infrastructure"